### ISSUE TITLE
Librarie (new Dynamo 2.0 Library) easily crashes WebGL

### ISSUE THREAD

**Issue by alfarok** [alfarok/DynamoViewport#4]
This is an issue with Librarie and not Viewport.  The current alternative is to exclusively use in-canvas search and minimize the left-hand Dynamo library window prior to launching.

**Comment by alfarok** [comment]
Can't reproduce on windows desktop, so far just when running Parallels on Mac.

**Comment by mjkkirschner** [comment]
this might happen when the resolution changes, which occurs often when parallels windows changes size- this causes the graphics adapter to disconnect and reconnect.

**Comment by alfarok** [comment]
@mjkkirschner yes, it must be something in that realm.  I actually haven't been able to crash it at all from library interaction which makes me very happy.  Now I just need you to review a PR enabling webGL by default ;)
